go.mod

```diff
@@ -4,11 +4,9 @@ go 1.12
 
 require (
 	github.com/alexflint/go-arg v1.3.0
-	github.com/operator-framework/operator-sdk v0.19.0
+	github.com/operator-framework/operator-sdk v0.5.0
```

Comment:
I'm not sure about this. Unfortunately, to work with OS3.11 we need version <=0.8.0. What about use 0.8.0 and try to update the che-operator operator-sdk verison later?
@tolusha do you see any blocker on che-operator side to not update operator-sdk to 0.8.0?

Reply:
if it's not a hard technical constrain I would prefer to use the latest operator-sdk in a separate task/pr